src/transaction/transaction_group_complete_manager.hpp

```diff
@@ -37,7 +37,9 @@ namespace cubtx
     GROUP_CLOSED = 0x01, /* Group closed. No other transaction can be included in a closed group. */
     GROUP_MVCC_COMPLETED = 0x02, /* MVCC completed. */
     GROUP_LOGGED = 0x04, /* Group log added. */
-    GROUP_COMPLETED = 0x08  /* Group completed. */
+    GROUP_PREPARED_FOR_COMPLETE = 0x08,  /* Group prepared for complete. */
```

Comment:
a short explanation of enum is required : these values are bit fields, but they also may be regarded as states
(setting of a specific bit assumes existence of a previous state).